src/app/editors/contentlist/contentlist.component.ts

```diff
@@ -81,8 +81,7 @@ export class ContentlistComponent implements OnInit {
     }
    };
     this.helperService.createContent(requestData).subscribe(res => {
-      console.log(res.result.identifier);
-      this.openContent(res.result.identifier);
+      this.openContent(res.result.content_id);
```

Comment:
"identifier" was right. Why did you changed now?

Reply:
@vinukumar-vs After changing the API, We are getting `content_id` in the response.